src/test/groovy/net/anshulverma/gradle/estilo/checkstyle/checks/FileLoaderTest.groovy

```diff
@@ -50,6 +52,9 @@ class FileLoaderTest extends AbstractSpecification {
       checkType        | hash
       CheckType.GOOGLE | [44, -55, 53, -59, -34, -78, 95, -46, -92, 29, 109, -104, 126, -12, 120, -16]
       CheckType.SUN    | [123, -62, -28, 57, 21, -38, -118, -78, -54, -54, -44, 67, -21, 63, -66, -79]
+      CheckType.EMPTY  | hex2byte("3e9f1a5f8e8d821e443ead6b7daeeed6")
+
+    CheckType.setCustom("src/main/resources/empty_checks.xml") | hex2byte("3e9f1a5f8e8d821e443ead6b7daeeed6")
```

Comment:
Testing out the custom config loading by just re-using the empty file but specifying using the `setCustom` method instead of using one of the pre-defined fields.